sources/web/datalab/polymer/components/table-details/table-details.html

```diff
@@ -0,0 +1,116 @@
+in compliance with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License
+is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+or implied. See the License for the specific language governing permissions and limitations under
+the License.
+-->
+
+<link rel="import" href="../../components/shared-styles/shared-styles.html">
+
+<link rel="import" href="../../bower_components/paper-spinner/paper-spinner.html">
+
+<dom-module id="table-details">
+  <template>
+    <style include="datalab-shared-styles">
+      #placeholder {
+        height: 100%;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+      paper-spinner {
+        --paper-spinner-stroke-width: 2px;
+      }
+      #container {
+        padding: 0px 20px;
+        height: 100%;
+        overflow-y: auto;
+        color: var(--primary-fg-color);
+      }
+      .strong {
+        font-weight: bold;
+      }
+      table {
+        border-spacing: 1px;
+        padding: 0 0 4px 0;
+        width: 100%;
+      }
+      td.field {
+        padding: 10px;
+        background-color: #eee;
+        font-weight: bold;
+      }
+      td.value {
+        padding: 10px;
+        background-color: #fff;
+      }
+      .separator {
+        margin: 20px 0px;
+      }
+    </style>
+    <div id="placeholder" hidden$={{_table}}>
+      <paper-spinner active hidden$={{!_busy}}></paper-spinner>
+    </div>
+    <div id="container" hidden$={{!_table}}>
+      <h3>Table Details: {{_table.tableReference.tableId}}</h3>
+      <span class="strong">Project: </span><span>{{_table.tableReference.projectId}}</span>
+      <br>
+      <br>
+      <span class="strong">Dataset: </span><span>{{_table.tableReference.datasetId}}</span>
+      <br>
+      <br>
+      <table>
+        <tr>
+          <td class="field">Table Size</td>
+          <td class="value">{{tableSize}}</td>
+        </tr>
+        <tr>
+          <td class="field">Long Term Storage Size</td>
+          <td class="value">{{longTermTableSize}}</td>
+        </tr>
+        <tr>
+          <td class="field">Number of Rows</td>
+          <td class="value">{{numRows}}</td>
+        </tr>
+        <tr>
+          <td class="field">Creation Time</td>
+          <td class="value">{{creationTime}}</td>
+        </tr>
+        <tr>
+          <td class="field">Last Modified</td>
+          <td class="value">{{lastModifiedTime}}</td>
+        </tr>
+        <tr>
+          <td class="field">Data Location</td>
+          <td class="value">{{_table.location}}</td>
+        </tr>
+      </table>
+
+      <hr class="separator">
+
+      <div class="strong">Schema</div>
+      <br>
+
+      <table>
+        <!--TODO: Flatten out the fields list-->
```

Comment:
Or maybe use an expander for nested fields, especially if there is a top-level type name that you can display for the group of fields.